### PR TITLE
Content guide

### PR DESCRIPTION
**Pull request by jesusbotella**
Content repository guide for Airship Developer Center.

https://github.com/CartoDB/airship/issues/235

### CODE REVIEW

File: docs/guides/02-content.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Airship Content
+
+Airship is divided into three main packages:
+
+- @carto/airship-components: Web components ready to use in your application.
```
Comment (IagoLast) [inline docs/guides/02-content.md]: Transform those items into a links to the npm packages

Comment (IagoLast) [reply docs/guides/02-content.md]: Package descriptions should be consistent all around the docs. We should agree on the right one

File: docs/guides/02-content.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- @carto/airship-style: CSS classes to provide the foundation for your application's look and feel.
+- @carto/airship-icons: Icon set to use in your HTML.
+
+These three packages are what Airship provides to build your own location intelligence application.
+
+## Components
+Airship components are built on top of StencilJS, which allows us to provide lightweight and easy to use components.
```
Comment (IagoLast) [inline docs/guides/02-content.md]: Link to StencilJS ?

File: docs/guides/02-content.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- Example HTML file: Component showcase showing all the component variants and behaviour.
+
+Airship components are always lazy loaded, so there is no need to worry about importing the whole bundle or each component separately.
+By importing `@carto/airship-components` package, the components loader will be included in your bundle and it will lazy-load components by injecting them as soon as they are present in the DOM.
+
+## Styles
+Airship styles are split into several folders to allow importing the whole bundle or each style separately, aiming to give flexibility and the possibility to reduce the size of the application's final bundle.
+
+**Styles are provided in two different flavours:**
+- **SASS**: raw uncompiled styles to process on the build process, allowing you to modify variables and customize Airship styles.
```
Comment (IagoLast) [inline docs/guides/02-content.md]: Capitalize first word in the descriptions

File: docs/guides/02-content.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+- CSS: `@carto/airship-style/dist/{path}`
+- SASS: `@carto/airship-style/src/{path}`
+
+In case you want to import the whole bundle, you need to import:
+- JavaScript: `import '@carto/airship-style';`
+- CSS: `@import '~@carto/airship-style';`
+
+If you want to include any of the styles separately, you need to look for the path and append it to the root directory folder.
+
+As of now, we have 16 style components as well as some core styles related to application layout. You can find all of them [here](https://github.com/CartoDB/airship/tree/master/packages/styles/src).
```
Comment (IagoLast) [inline docs/guides/02-content.md]: > As of now, we have 16 style components 

This is going to be outdated very often, I suggest:

> You can find a list of CSS elements [here](https://github.com/CartoDB/airship/tree/master/packages/styles/src)

File: docs/guides/02-content.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  - JavaScript import: `import '@carto/airship-style/dist/badges/badges';`
+  - SASS import: `@import '~@carto/airship-style/dist/badges/badges';`
+
+## Icons
+Airship includes a set of icons to place into your application. These icons can be found in [this folder](https://github.com/CartoDB/airship/tree/master/packages/icons/src/icons).
+
+Each SVG is automatically generated from a path set in [`paths.js`](https://github.com/CartoDB/airship/blob/master/packages/icons/src/paths.js).
+
+Similarly to styles, we provide icons in two flavours:
+- **SVG files**: plain SVG files to include within an `<img>` tag.
```
Comment (IagoLast) [inline docs/guides/02-content.md]: Capitalize first letter in the description

File: docs/guides/02-content.md (hunk 1 above)
Comment (jesusbotella) [reply docs/guides/02-content.md]: Yes, we need to give it a look and decide the official descriptions.

**Pull request merged**